src/coreclr/jit/assertionprop.cpp

```diff
@@ -83,6 +83,7 @@ bool IntegralRange::Contains(int64_t value) const
     {
         case TYP_UBYTE:
         case TYP_USHORT:
+        case TYP_UINT:
```

Comment:
I think the previous comment on this got resolved, but I wouldn't expect this to be necessary since we shouldn't be encountering any nodes where `gtType` is `TYP_UINT`.

Reply:
Yes, I have removed the dead code. We never hit this condition in the latest code.